packages/plugins/plugin-outliner/src/types/types.ts

```diff
@@ -32,11 +33,11 @@ export interface OutlineType extends Schema.Schema.Type<typeof OutlineType> {}
 
 export const JournalEntryType = Schema.Struct({
   date: Schema.String, // TODO(burdon): Date.
-  tree: Ref(TreeType),
+  content: Ref(DataType.Text), // TODO(burdon): Breaks unless this is a reference.
```

Comment:
Is there a reason that this is a separate type from `OutlineType`? Could `JournalType` entries just be `OutlineType`s?